pkg/virt-config/config-map.go

```diff
@@ -52,6 +52,8 @@ const (
 	MemoryOvercommitKey       = "memory-overcommit"
 	LessPVCSpaceTolerationKey = "pvc-tolerate-less-space-up-to-percent"
 	NodeSelectorsKey          = "node-selectors"
+	NetworkInterfaceKey       = "default-network-interface"
+	DisableSlirp              = "disableSlirp"
```

Comment:
For opt-in options, I'd typically expect to opt-in by enabling some feature that defaults to `false`.  Here we're opting in by disabling a feature that defaults to `true`. 

Would it be any more consistent for us to name the option something like `permitSlirpInterfaces` which defaults to `false`?

I don't care that much either way, but opting in by enabling something makes a little more sense to me.

Reply:
:+1: Thanks, `permitSlirpInterfaces` sounds better to me.

Reply:
Done :)